desktop/hardened-desktop/settings-management/configure.md

```diff
@@ -0,0 +1,139 @@
+---
+description: settings management for desktop
+keywords: admin, controls, rootless, enhanced container isolation
+title: Configure Settings Management
+--- 
+
+>**Note**
+>
+>Settings Management is available to Docker Business customers only. 
+
+This page contains information for admins on how to configure Settings Management to specify and lock configuration parameters to create a standardized Docker Desktop environment across the organization.
+
+Settings Management is designed specifically for organizations who don’t give developers root access to their machines.
+
+### Prerequisites
+
+- [Download and install Docker Desktop 4.13.0 or later](../../release-notes.md).
+- As an admin, you need to [configure a registry.json to enforce sign-in](../../../docker-hub/configure-sign-in.md). This is because this feature requires a Docker Business subscription and therefore your Docker Desktop users must authenticate to your organization for this configuration to take effect.
+
+### Step one: Create the `admin-settings.json` file and save it in the correct location
+
+1. Create a new, empty JSON file and name it `admin-settings`.
+2. Save the `admin-settings.json` file on your developers' machines in the following locations:
+
+    - Mac: `/Library/Application\ Support/com.docker.docker/admin-settings.json`
+    - Windows: `/ProgramData/DockerDesktop/admin-settings.json`
```

Comment:
On Windows paths use "\" instead of "/". Please change to:

Windows: `C:\ProgramData\DockerDesktop\admin-settings.json`